src/gitManager/gitManager.ts

```diff
@@ -123,8 +124,12 @@ export abstract class GitManager {
     // @param doConversion - If false, the path is returned as is. This is added because that parameter is often passed on to functions where this method is called.
     getRelativeRepoPath(path: string, doConversion: boolean = true): string {
         if (doConversion) {
-            if (this.plugin.settings.basePath.length > 0) {
-                return path.substring(this.plugin.settings.basePath.length + 1);
+            let adapter = this.plugin.app.vault.adapter;
+            if (adapter instanceof FileSystemAdapter) {
```

Comment:
On mobile, this method does nothing now. I think we should fall back to the old implementation on mobile.